### PR TITLE
Dịch "fashion-mnist_vn" - Phần 2

### LINKED ISSUE #757: Dịch "fashion-mnist_vn" - Phần 2

### PR DESCRIPTION
**Pull request by quangnhat185**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #757 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 1]
```diff
@@ -114,14 +114,15 @@ show_images(X.squeeze(axis=-1), 2, 9, titles=get_fashion_mnist_labels(y));
...
+## Đọc một Minibatch
 
 <!--
 To make our life easier when reading from the training and test sets, we use a `DataLoader` rather than creating one from scratch, as we did in :numref:`sec_linear_scratch`.
 Recall that at each iteration, a `DataLoader` reads a minibatch of data with size `batch_size` each time.
 -->
 
-*dịch đoạn phía trên*
+Để dễ dàng hơn trong việc đọc dữ liệu từ tập huấn luyện và tập kiểm tra, chúng ta sử dụng một `DataLoader` có sẵn thay vì tạo từ đầu như đã làm ở :numref:`sec_linear_scratch`.
+Nhắc lại là ở mỗi vòng lặp, một `DataLoader`sẽ đọc một minibatch của bộ dữ liệu với kích thước `batch_size`.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Nhắc lại là ở mỗi vòng lặp, một `DataLoader` sẽ đọc một minibatch của bộ dữ liệu với kích thước `batch_size`.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 2]
```diff
@@ -130,7 +131,10 @@ For instance, we can set aside 4 processes to read the data (via `num_workers`).
 Because this feature is not currently supported on Windows the following code checks the platform to make sure that we do not saddle our Windows-using friends with error messages later on.
 -->
 
-*dịch đoạn phía trên*
+Trong quá trình huấn luyện, việc đọc dữ liệu có thể trở thành một nút cổ chai về hiệu năng đáng kể, trừ khi chúng ta sử dụng một mô hình đơn giản hoặc là máy tính của chúng ta rất nhanh.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Trong quá trình huấn luyện, việc đọc dữ liệu có thể trở thành một nút cổ chai đáng kể về hiệu năng, trừ khi chúng ta sử dụng một mô hình đơn giản hoặc là máy tính của chúng ta rất nhanh. 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 3]
```diff
@@ -149,7 +153,10 @@ The `ToTensor` class also moves the image channel from the last dimension to the
 Through the `transform_first` function of the dataset, we apply the transformation of `ToTensor` to the first element of each instance (image and label).
 -->
 
-*dịch đoạn phía trên*
+Đưới đây, chúng ta chuyển đổi dữ liệu hình ảnh từ uint8 sang số thực động (_floating point number_) 32-bit với class `ToTensor`.
+Ngoài ra, bộ chuyển đổi sẽ chia tất cả các số với 255 để tất cả các pixels sẽ có giá trị từ 0 đến 1. 
+Class `ToTensor` cũng chuyển kênh hình ảnh từ chiều cuối cùng sang chiều thứ nhất để tạo điều kiện cho các tính toán mạng nơ-ron sẽ được giới thiệu sau này.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Lớp `ToTensor` cũng chuyển kênh hình ảnh từ chiều cuối cùng sang chiều thứ nhất để tạo điều kiện cho các tính toán mạng nơ-ron sẽ được giới thiệu sau này.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 4]
```diff
@@ -149,7 +153,10 @@ The `ToTensor` class also moves the image channel from the last dimension to the
...
+Thông qua hàm `transform_first` của bộ dữ liệu, chúng ta có thể áp dụng phép biến đổi `ToTensor` cho phần tử đầu tiên  của mỗi ví dụ (ảnh và nhãn).
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Thông qua hàm `transform_first` của bộ dữ liệu, chúng ta có thể áp dụng phép biến đổi `ToTensor` cho phần tử đầu tiên của mỗi ví dụ (ảnh và nhãn). 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 5]
```diff
@@ -163,7 +170,7 @@ train_iter = gluon.data.DataLoader(mnist_train.transform_first(transformer),
 Let's look at the time it takes to read the training data.
 -->
 
-*dịch đoạn phía trên*
+Cùng xem thời gian cần thiết để hoàn tất việc đọc và huấn luyện dữ liệu.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Hãy cùng xem thời gian cần thiết để hoàn tất việc đọc dữ liệu huấn luyện.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 6]
```diff
@@ -149,7 +153,10 @@ The `ToTensor` class also moves the image channel from the last dimension to the
 Through the `transform_first` function of the dataset, we apply the transformation of `ToTensor` to the first element of each instance (image and label).
 -->
 
-*dịch đoạn phía trên*
+Đưới đây, chúng ta chuyển đổi dữ liệu hình ảnh từ uint8 sang số thực động (_floating point number_) 32-bit với class `ToTensor`.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Dưới đây, chúng ta chuyển đổi dữ liệu hình ảnh từ uint8 sang số thực dấu phẩy động (_floating point number_) có 32-bit với lớp `ToTensor`.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 7]
```diff
@@ -149,7 +153,10 @@ The `ToTensor` class also moves the image channel from the last dimension to the
...
+Ngoài ra, bộ chuyển đổi sẽ chia tất cả các số với 255 để tất cả các pixels sẽ có giá trị từ 0 đến 1.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Ngoài ra, bộ chuyển đổi sẽ chia tất cả các số với 255 để tất cả các điểm ảnh sẽ có giá trị từ 0 đến 1. 
```

File: chapter_linear-networks/fashion-mnist_vn.md (hunk 2 above)
Comment (aivivn) [reply chapter_linear-networks/fashion-mnist_vn.md]: `bottlenech` dịch là `nút thắt cổ chai` hoặc `thắt cổ chai` hợp lý hơn. Từ `thắt` quan trọng hơn từ `nút`.

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 8]
```diff
@@ -293,11 +300,10 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 2 -->
-*
+* Nguyễn Lê Quang Nhật
```
Comment (quangnhat185) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
* Nguyễn Lê Quang Nhật
* @rootonchair 
* @aivivn 
* @duythanhvn 
```

Comment (duythanhvn) [reply chapter_linear-networks/fashion-mnist_vn.md]: Dùng tên thật thôi ông giáo ơi, có tên hết rồi nờ.

Ps: tên của aivivn là "trùm cuối"....

Comment (tiepvupsu) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
* Nguyễn Lê Quang Nhật
* Phạm Hồng Vinh
```

Comment (quangnhat185) [reply chapter_linear-networks/fashion-mnist_vn.md]: > Dùng tên thật thôi ông giáo ơi, có tên hết rồi nờ.
> 
> Ps: tên của aivivn là "trùm cuối"....

Có nên để là the  FINAL BOSS ko nhỉ :)))
